Makefile

```diff
@@ -5,29 +5,31 @@ data/external/deep/glove.840B.300d.txt:
 	rm /tmp/glove.840B.300d.zip
 
 output/deep/params: data/external/deep/glove.840B.300d.txt
-	python3 guesser/util/format_dan.py --threshold=5
-	python3 guesser/util/load_embeddings.py
-	python3 guesser/dan.py
+	mkdir -p data/external/deep/train
+	python3 qanta/guesser/util/format_dan.py --threshold=5
+	python3 qanta/guesser/util/load_embeddings.py
+	python3 qanta/guesser/dan.py
 
 output/kenlm.binary:
 	mkdir -p temp
-	python3 cli.py build_mentions_lm_data data/external/wikipedia /tmp/wiki_sent
+	python3  cli.py build_mentions_lm_data data/external/wikipedia /tmp/wiki_sent
 	lmplz -o 5 < /tmp/wiki_sent > output/kenlm.arpa
 	build_binary output/kenlm.arpa $@
 	rm /tmp/wiki_sent
 
-output/wikifier/data/input/:
+output/wikifier/data/input:
 	rm -rf $@
 	mkdir -p $@
 	python3 qanta/wikipedia/wikification.py
 
-output/wikifier/data/output/: output/wikifier/data/input
+output/wikifier/data/output: output/wikifier/data/input
 	rm -rf $@
 	mkdir -p $@
 	cp lib/STAND_ALONE_NO_INFERENCE.xml output/wikifier/STAND_ALONE_NO_INFERENCE.xml
 	mkdir -p output/wikifier/data/configs
 	cp lib/jwnl_properties.xml output/wikifier/data/configs/jwnl_properties.xml
-	(cd output/wikifier && java -Xmx10G -jar wikifier-3.0-jar-with-dependencies.jar -annotateData data/input data/output false STAND_ALONE_NO_INFERENCE.xml)
+	cp lib/wikifier-3.0-jar-with-dependencies.jar output/wikifier/data/configs/wikifier-3.0-jar-with-dependencies.jar
```

Comment:
Using `diff` it seems like the config files are identical so instead letting wikifier use its own.